src/test/java/nl/tudelft/lifetiles/graph/models/jgrapht/GraphAdapterTest.java

```diff
@@ -33,8 +33,9 @@ public static void runOnce() {
     @Before
     public void setUp() throws Exception {
         gf = fp.getFactory("JGraphT");
-        v1 = new SequenceSegment(null, 0, 0, null);
-        v2 = new SequenceSegment(null, 0, 0, null);
+        v1 = new SequenceSegment(null, 1, 5, null);
+        v2 = new SequenceSegment(null, 6, 10, null);
+        v3 = new SequenceSegment(null, 7, 10, null);
         gr = gf.getGraph();
```

Comment:
I disagree, we should just test the JGraphTGraphAdapter in this test without adding additional dependencies on the parser.